dcs/src/main/java/org/trafodion/dcs/master/listener/ConnectReply.java

```diff
@@ -275,12 +275,10 @@ boolean buildConnectReply  (Header hdr, ConnectionContext cc, SocketAddress clie
         if (found == false || exceptionThrown == true){
             exception.exception_nr = ListenerConstants.DcsMasterNoSrvrHdl_exn; //no available servers
             replyException = true;
-            if(LOG.isDebugEnabled()){
-                if (found == false)
-                    LOG.info(clientSocketAddress + ": " + "No Available Servers");
-                else
-                    LOG.info(clientSocketAddress + ": " + "No Available Servers - exception thrown");
-            }
+            if (found == false)
```

Comment:
It looks like both (found == false) and (exceptionThrown == true) could be true. If they are both true, you will log "No Available Servers", but won't record that an exception is thrown. Is this what you want?